odb.spec

```diff
@@ -0,0 +1,29 @@
+### RPM external odb 2.5.0-b.3
+Source0: https://codesynthesis.com/~boris/tmp/odb/pre-release/b.3/odb-2.5.0-b.3.tar.gz
```

Comment:
this is a particularly interesting place to get source code from. What is this thing?
(added bonus, GPLv3. naively -1)

Reply:
ODB is from codesynthesis. It is an object releational database. The odb command generates database code based on a model. CodeCompass uses this to interface with a database that it generates.

Reply:
CodeCompass is something Chris asked to look into earlier this year.
https://github.com/Ericsson/CodeCompass/blob/master/doc/deps.md